lib/utils.js

```diff
@@ -30,4 +33,31 @@ function configureBinaryLog (opts) {
   log.level = opts.debug ? 'debug' : 'info';
 }
 
-export { pkgRoot, ok, nok, inquirer, configureBinaryLog, authorizeIos, };
+/**
+ * Return an executable path of cmd
+ *
+ * @param {string} cmd Standard output by command
+ * @return {?string} The full path of cmd. `null` if the cmd is not found.
+ */
+async function resolveExecutablePath (cmd) {
+  const executable = system.isWindows() ? 'where' : 'which';
+  let stdout;
+  try {
+    stdout = (await exec(executable, [cmd])).stdout;
+    if (stdout.match(/not found/gi)) {
+      throw new Error('Not Found');
```

Comment:
what is the point to throw an error and then catch it?

Reply:
I figure out windows returns `not found` message when the command cannot find it.
But the message is localised one. It works only for English environment.

I'll change like below

```
if (!stdout.includes(cmd)) {
    return null;
}
```